Tasks/Ryabov/task2/task2.py

```diff
@@ -0,0 +1,20 @@
+d={9103976271:[("Reina", "Meinhard"), ("Memphis", "Tennessee")],
+4199392609:[("Stephanie", "Bruce"), ("Greensboro", "North Carolina")],
+9099459979:[("Ermes", "Angela"), ("Dallas", "Texas")],
+6123479367:[("Lorenza", "Takuya"), ("Indianapolis", "Indiana")],
+7548993768:[("Margarete", "Quintin"), ("Raleigh", "North Carolina")]}
+print(d)
+
+x = int(input("Enter phone namber"))
```

Comment:
where is some validation? it was specified in the task description, that we need to check the user input at least for the length and digits

-15 points for the missing part of task